src/dmd/todt.d

```diff
@@ -33,6 +33,7 @@ import dmd.errors;
 import dmd.expression;
 import dmd.func;
 import dmd.globals;
+import dmd.glue;
```

Comment:
Is this a result of fixing the bug?

Reply:
Well, it's to fix a instance of the bug in DMD.

Reply:
There are two lines a little below:
```
alias toSymbol = dmd.tocsym.toSymbol;
alias toSymbol = dmd.glue.toSymbol;
```
`dmd.glue` is not imported but is working because of the bug.